meson.build

```diff
@@ -188,12 +188,15 @@ endif
 bison_cmd = [bison, bison_flags, '-o', '@OUTPUT0@']
 
 flex_flags = []
-flex_cmd = [flex, '-o', '@OUTPUT0@']
+flex_wrapper = files('src/tools/pgflex')
+flex_cmd = [python, flex_wrapper, '--rootdir', meson.build_root(), '--flex', flex, '-o', '@OUTPUT0@']
 
 # FIXME: --backup-file is only available in unreleased flex (used in
 # Debian). Without it -b just puts lex.backup into CWD - with ninja that's the
 # root build directory. So we need to do all of this in a temporary directory...
```

Comment:
I changed working directory in meson wrapper script, so I think it is creating lex.backup files in correct paths now. It was creating 4 lex.backup files at the .l files' paths. Does that solve that comment?

Reply:
Not quite - there's directories with two .l files, so I think we need to use to a temporary directory to avoid conflicts. If you pass @PRIVATE_DIR@ as an argument, meson will replace that with the name of a temporary directory.

Reply:
Cool. Now the FIXME below your changes can be removed, right?

Reply:
Done.